app/src/components/TokenCard.tsx

```diff
@@ -0,0 +1,23 @@
+
+import React from 'react';
+
+import { Box, Direction, Image, PaddingSize, Spacing, Stack, Text, TextAlignment } from '@kibalabs/ui-react';
+
+export const TokenCard = (): React.ReactElement => {
+  return (
+    <Stack direction={Direction.Vertical} isFullHeight={true} isFullWidth={true} padding={PaddingSize.Wide}>
+      <Text variant='header3-singleLine'>Your Holdings</Text>
+      <Spacing variant={PaddingSize.Wide} />
+      <Box variant='card' height='204px' width='140px'>
```

Comment:
these sizes should definitely not be hardcoded from the start, try to not hardcode them and only add the hardcoding right at the end if nothing works